packages/caver-klay/caver-klay-kct/contract/introspection/IKIP13.sol

```diff
@@ -0,0 +1,22 @@
+pragma solidity ^0.5.0;
+
+/**
+ * @dev Interface of the KIP13 standard, as defined in the
+ * [KIP](https://klaytn.github.io/kips/KIPs/kip-interface_query_standard).
+ *
+ * Implementers can declare support of contract interfaces, which can then be
+ * queried by others (`KIP13Checker`).
```

Comment:
Where can we find KIP13Checker?

Reply:
I think the KIP13Checker in this description means supportsInterface function.

Reply:
Do we have any definition of this somewhere?

Reply:
There seems to be no definition for KIP13Checker.
I will remove ambiguous explanations.